packages/lib/services/searchengine/SearchEngine.ts

```diff
@@ -580,14 +575,32 @@ export default class SearchEngine {
 			// https://github.com/laurent22/joplin/issues/1075#issuecomment-459258856
 
 			try {
-				const { query, params } = queryBuilder(parsedQuery.allTerms);
+				const { query, params } = queryBuilder(parsedQuery.allTerms, true);
 				const rows = await this.db().selectAll(query, params);
 				this.processResults_(rows, parsedQuery);
 				return rows;
 			} catch (error) {
 				this.logger().warn(`Cannot execute MATCH query: ${searchString}: ${error.message}`);
 				return [];
 			}
+		} else {
+			searchString = this.normalizeText_(searchString);
+
+			let rows: any;
```

Comment:
Shouldn't it be at any[]? And any chance you could define an interface for it? You don't have to apply it everywhere, just in the code you've added here. I guess it's just something like `interface Row { title: string; body: string; etc. }` ?

Reply:
I removed this variable, do you think the interface is still needed?

Reply:
No, no need to worry about this. Even better if removed the variable.